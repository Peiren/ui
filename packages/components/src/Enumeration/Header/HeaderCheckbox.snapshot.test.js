import React from 'react';
import renderer from 'react-test-renderer';

import HeaderCheckbox from './HeaderCheckbox.component';

describe('HeaderCheckbox', () => {
	it('should display toggle for mass selecting', () => {
		// given
		const props = {
			headerDefault: [],
		};

		// when
		const wrapper = renderer.create(
			<HeaderCheckbox {...props}/>
		).toJSON();

		// then
		expect(wrapper).toMatchSnapshot();
	});
});
