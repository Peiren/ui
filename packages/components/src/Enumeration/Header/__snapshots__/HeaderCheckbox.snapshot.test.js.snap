// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderCheckbox should display toggle for mass selecting 1`] = `
<header
  className="tc-enumeration-header"
>
  <div
    className="checkbox"
  >
    <label
      htmlFor="toggle-all"
    >
      <input
        checked={false}
        id="toggle-all"
        onChange={undefined}
        type="checkbox"
      />
      <strong>
        Toggle all
      </strong>
    </label>
  </div>
</header>
`;
